README.md

```diff
@@ -78,3 +78,26 @@ return [
     ],
 ];
 ```
+
+**SSL configuration with self-signed certificates** NOTE: This has security implications.
```

Comment:
We should not encourage such configurations.

Reply:
I think it's important that the options are documented in the README.md but maybe you are right that it should include all configuration options with their default values instead of throwing this kind of use case to your face. Do you want to refactor the README to match your idea or should I try again?

Reply:
I would remove this paragraph and just add a line in the example above it mentioning new property (or just not add anything at all).

Reply:
I'll adjust the README based on this.